contracts/gauge_controller.sol

```diff
@@ -264,6 +278,7 @@ contract GaugeController is AccessControlUpgradeable, ReentrancyGuardUpgradeable
      */
     function voteForGaugeWeight(address _gAddr, uint256 _userWeight)
         external
+        whenNotPaused
```

Comment:
Please remove the "whenNotPaused" modifier from the voteForGaugeWeight() function. If we make the voteForGaugeWeight function pausable, users might not be able to utilize their already locked voting power from the VotingEscrow contract, resulting in a situation where they can lock but cannot vote.